src/app/space/add-space-overlay/add-space-overlay.component.html

```diff
@@ -1,64 +1,96 @@
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>
+<div class="modal-body container-fluid f8-add-space-overlay">
+  <div class="row f8-add-space-overlay--container">
+    <section class="col-sm-6 f8-add-space-overlay--left-section" [ngClass]="{'has-error': name.errors && name.dirty}">
+        <form #spaceForm="ngForm">
+          <div class="alert alert-danger alert-dismissable"
+            *ngIf="name.errors && name.dirty">
+            <span class="pficon pficon-error-circle-o"></span>
+            <span [hidden]="!name.errors.required">Space Name is required to create a Space.</span>
+            <span [hidden]="!name.errors.minLength">{{name.errors.minLength?.message}}</span>
+            <span [hidden]="!name.errors.maxLength">{{name.errors.maxLength?.message}}</span>
+            <span [hidden]="!name.errors.unique">{{name.errors.unique?.message}}</span>
+            <span [hidden]="!name.errors.invalid">{{name.errors.invalid?.message}}</span>
+          </div>
+          <div class="col-sm-9">
+            <label class="font-size-small f8-add-space-overlay-label" for="add-space-overlay-name">
+              <span class="required">*</span>
+              Name
+            </label>
+            <input id="add-space-overlay-name" class="form-control" name="name"
+              type="text" placeholder="Enter a unique name for your space"
+              #addSpaceOverlayNameInput
+              [(ngModel)]="space.name" #name="ngModel" required uniqueSpaceName validSpaceName
+              (keyup.enter)="createSpaceButton.click()" autocomplete="off"/>
+            <p class="font-size-small f8-add-space-overlay--tip help-block">Names must contain 4-40 characters, contain only letters, numbers, underscores
+              or hyphens, not start with a number, and not start or end with underscores or hyphens.
+            </p>
+            <label class="font-size-small f8-add-space-overlay-label" for="add-space-overlay-description">
+              Description
+            </label>
+            <input id="add-space-overlay-description" class="form-control"
+              name="description" type="text" placeholder="Enter a Space Description" #description/>
           </div>
         </form>
       </section>
-      <footer class="container-fluid padding-top-standard-offset padding-bottom-standard-offset">
-        <div class="row">
-          <div class="col-sm-12 create-launcher-step-tool-bar button-right">
-            <button #createSpaceButton id="createSpaceButton" class="btn btn-primary" [disabled]="!spaceForm.form.valid || !canSubmit" type="submit">Ok</button>
-            <button id="cancelSpaceButton" class="btn btn-link" aria-label="Cancel"
-                    (click)="hideAddSpaceOverlay()">Cancel</button>
+      <section class="col-sm-6 f8-add-space-overlay--right-section">
+        <div class="col-sm-9">
```

Comment:
`col-sm-9` adds too much dead space on the right. We shouldn't constrain the text at all here. Remove.